www/layouts/partials/home_course_collections.html

```diff
@@ -6,7 +6,7 @@ <h2>Discover Collections</h2>
       collections. Below are some topics available for you to explore:
     </p>
     <div class="course-collection-section d-flex flex-row m-0 flex-wrap">
-      {{ range where .Site.Pages "Type" "course_collections" }}
+      {{ range where .Site.Pages "Type" "course_collections" | intersect (where .Site.Pages "Title" "in" .Params.homePageCourseCollections)}}
```

Comment:
These are page now, not course collections. Sorry for the confusing naming, but we want the home to link to pages, and the pages will link to course collections. 

On production, the relevant URLs are:




title | production URL (although partial URLs would be better) 
-- | --
Open Learning Library | https://ocwnext.odl.mit.edu/pages/open-learning-library/
Introductory Programming | https://ocwnext.odl.mit.edu/pages/introductory-programming/
Energy | https://ocwnext.odl.mit.edu/pages/energy/
Entrepreneurship | https://ocwnext.odl.mit.edu/pages/entrepreneurship/
Environment | https://ocwnext.odl.mit.edu/pages/environment/
Transportation | https://ocwnext.odl.mit.edu/pages/transportation/

Reply:
I updated  the partial to link to pages. Sorry for the delay, I misunderstood the issue